core-schema/src/main/resources/sql-schema/V6__Mod_kitpvp.sql

```diff
@@ -4,6 +4,9 @@ CREATE TABLE kitpvp_statistics (
   kills INT NOT NULL,
   deaths INT NOT NULL,
   assists INT NOT NULL,
+  current_killstreak INT NOT NULL,
```

Comment:
Are you sure we really need to persist this value in the database? I'd expect it to be in-memory only, as once a player switches servers, their killstreak can't continue, can it?

Reply:
If you have played kitpvp before i do believe that "current killstreak" persists between servers